source/MaterialXFormat/XmlIo.cpp

```diff
@@ -191,7 +191,9 @@ void processXIncludes(DocumentPtr doc, xml_node& xmlNode, const string& searchPa
                 readXIncludeFunction(library, filename, includeSearchPath, &xiReadOptions);
 
                 // Import the library document.
-                doc->importLibrary(library);
+                CopyOptions copyOptions;
```

Comment:
This option got removed in master so making it skip all the time for now until ILM has a better logic for this as discussed in the slack channel.